codemagic.yaml

```diff
@@ -47,7 +47,7 @@ workflows:
     name: Build Documentation
     max_build_duration: 60
     environment:
-      xcode: 15.3
+      xcode: 16.0
```

Comment:
Wait, it just works now? Without changing anything in the build docs pipeline? How? 😄 

If this works, then we can also close this issue: https://github.com/tuist/tuist/issues/6775

Reply:
I [adjusted](https://github.com/tuist/tuist/pull/6843/files#diff-dd95957e8b7c35736f14845030c9fab97f2fe426900040df3777a40bf35deb20) the command that we use to dump the manifest to run in a temporary directory. The failure happened because the directory we were running the command from tried to load the `Tuist/Config.swift`, which is unnecessary for dumping the manifest, and that failed. Why it failed is something we'd still need to investigate, but since we haven't seen such reports, I'd keep it as low priority.